apps/mudita-center-e2e/src/specs/stress-tests/connected-devices-stress-test.ts

```diff
@@ -0,0 +1,334 @@
+import { E2EMockClient } from "../../../../../libs/e2e-mock/client/src"
+import {
+  overviewDataWithOneSimCard,
+  overviewDataWithOneSimCard2nd,
+  overviewDataWithOneSimCard3rd,
+  overviewDataWithOneSimCard4th,
+  overviewDataWithOneSimCard5th,
+  overviewDataWithOneSimCard6th,
+} from "../../../../../libs/e2e-mock/responses/src"
+import OverviewKompaktPage from "../../page-objects/overview-kompakt.page"
+import HomePage from "../../page-objects/home.page"
+import { kompaktImageRegex } from "../../consts/regex-const"
+import overviewPage from "../../page-objects/overview.page"
+import drawerPage from "../../page-objects/drawer.page"
+
+describe("Kompakt switching devices", () => {
+  const firstSerialNumber = "KOM1234567890"
+  const secondSerialNumber = "KOM1234567892"
+  const thirdSerialNumber = "KOM1234567893"
+  const fourthSerialNumber = "KOM1234567894"
+  const fifthSerialNumber = "KOM1234567895"
+  const sixthSerialNumber = "KOM1234567896"
+
+  before(async () => {
+    E2EMockClient.connect()
+    await browser.waitUntil(() => {
+      return E2EMockClient.checkConnection()
+    })
+  })
+
+  after(() => {
+    E2EMockClient.stopServer()
+    E2EMockClient.disconnect()
+  })
+
+  it("Connect 1,2 Kompakt devices", async () => {
+    const devices = [
+      {
+        path: "path-1",
+        body: overviewDataWithOneSimCard,
+        serialNumber: firstSerialNumber,
+      },
+      {
+        path: "path-2",
+        body: overviewDataWithOneSimCard2nd,
+        serialNumber: secondSerialNumber,
+      },
+    ]
+
+    for (const device of devices) {
+      E2EMockClient.mockResponse({
+        path: device.path,
+        body: device.body,
+        endpoint: "FEATURE_DATA",
+        method: "GET",
+        status: 200,
+      })
+      E2EMockClient.addDevice({
+        path: device.path,
+        serialNumber: device.serialNumber,
+      })
+
+      await browser.pause(6000)
+    }
+  })
+
+  it("Check Drawer Modal and Switch to 1st device", async () => {
+    const deviceSelectDrawer = await drawerPage.deviceSelectDrawer
+    await expect(deviceSelectDrawer).toBeDisplayed()
+
+    const deviceImageOnDrawer = await drawerPage.deviceImageOnDrawer
+    await expect(deviceImageOnDrawer).toBeDisplayed()
+
+    const firstDeviceOnDrawer = await drawerPage.getDeviceOnDrawer(
+      firstSerialNumber
+    )
+    const secondDeviceOnDrawer = await drawerPage.getDeviceOnDrawer(
+      secondSerialNumber
+    )
+    await expect(firstDeviceOnDrawer).toBeDisplayed()
+    await firstDeviceOnDrawer.waitForClickable()
+    await firstDeviceOnDrawer.click()
+  })
+  it("Verify Overview Page for 1st device", async () => {
+    const kompaktImage = await OverviewKompaktPage.kompaktImage
+    await expect(kompaktImage).toBeDisplayed()
+    await expect(kompaktImage).toHaveAttribute("src", kompaktImageRegex)
+
+    const kompaktOsVersion = await OverviewKompaktPage.kompaktOsVersion
+    await expect(kompaktOsVersion).toBeDisplayed()
```

Comment:
you may want to check what text contains `kompaktOsVersion` element

Reply:
same for devices below

Reply:
added await expect(kompaktOsVersion).toHaveText("Mudita OS") for each device